### PR TITLE
Use register interface for screen adjustment

### LINKED ISSUE #19: tv5725.h definition
Not sure how to use GIT yet, so I'm filling this file with register descriptions according to register map file, I'll upload it section by section starting from S3

### PR DESCRIPTION
**Pull request by bkoropoff**
No real functional change, just simplifies the code.

Closes #19 as well

### CODE REVIEW

File: gbs-control.ino [hunk 1]
```diff
@@ -1597,14 +1490,14 @@ void doPostPresetLoadSteps() {
   }
   if (rto->videoStandardInput == 3) {
     Serial.println(F("HDTV mode"));
-    scaleVerticalAbsolute(1023); // temporary
-    scaleHorizontalAbsolute(708); // temporary
+    GBS::VDS_VSCALE::write(1024); // temporary
```
Comment (ramapcsx2) [inline gbs-control.ino]: 1024?

**Comment by ramapcsx2** [comment]
Nice clutter reduction, maybe this even fixes a bug or two :)

File: gbs-control.ino (hunk 1 above)
Comment (bkoropoff) [reply gbs-control.ino]: Whoops

**Comment by bkoropoff** [comment]
Fixed

**Pull request merged**